DQM/DTMonitorModule/src/DTDataIntegrityTask.cc

```diff
@@ -603,86 +733,92 @@ void DTDataIntegrityTask::processuROS(DTuROSROSData& data, int fed, int uRos) {
     }
 
     if (uRos < 3) {
-      ROSSummary->Fill(tdcError_ROSSummary, link + 1);  //link 0 = ROS 1
-      int sc = 24;
-      if (mode <= 2) {
-        urosHistos[(uROSError)*1000 + (wheel + 2) * 100 + (link)]->Fill(tdcError_ROSError, sc);
-        if (mode <= 1)
-          urosHistos[(TDCError)*1000 + (wheel + 2) * 100 + (link)]->Fill(tdcError_TDCHisto + 6 * (tdc - 1),
-                                                                         sc);  // ros-1=link in this case
-      }                                                                        //mode<=2
-    }                                                                          //uRos<3
-    else {                                                                     //uRos>2
-      if (link < 24)
-        ROSSummary->Fill(tdcError_ROSSummary, ros);
-      else if (link < 48)
-        ROSSummary->Fill(tdcError_ROSSummary, ros + 1);
-      else if (link < 72)
-        ROSSummary->Fill(tdcError_ROSSummary, ros + 2);
-
-      if (mode <= 2) {
+      int sector = link + 1;
+      errorX[5][link][wheel + 2] += 1;
+      if (mode != 1) {
+        if (sector == 9)
+          sector = 10;
+        ROSSummary->Fill(tdcError_ROSSummary, sector - 1);  //link 0 = ROS 1
+        int ch25 = 24;
+        if (mode <= 2) {
+          urosHistos[(uROSError)*1000 + (wheel + 2) * 100 + (sector - 1)]->Fill(tdcError_ROSError, ch25);
+          if (mode <= 1)
```

Comment:
Given L738 above, this should be 
```suggestion
          if (mode < 1)
```